utils/createStrapi.js

```diff
@@ -27,12 +28,59 @@ const createStrapiProject = async () => {
 				active: `Yes`,
 				inactive: `No`,
 				type: `toggle`
+			},
+			{
+				name: `projectPath`,
```

Comment:
We can give this an intial value of the current `process.cwf()` I think 🤔 that way they don't need to type it all the time.

Reply:
made the change now the initial value has process.cwd()